README.md

```diff
@@ -1,34 +1,42 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Hololive EN Fan website
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+This repository holds the website for [holoen.fans](https://holoen.fans), which was started by the [Hololive EN Fan Discord](https://discord.gg/holoenfans).
 
-## Getting Started
+## Stack
 
-First, run the development server:
+* [Next.JS](https://nextjs.org) [React](https://reactjs.org/)
+* [MongoDB](http://mongodb.com/)
+* [Tailwind CSS](https://tailwindcss.com/)
 
-```bash
-npm run dev
-# or
-yarn dev
-```
+## Contributing
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+* Have NodeJS installed(obviously) and clone this repo
+* Copy `.env.example` to `.env.local` and fill all the values
+* You can now start coding! Start the dev server by running `npm run dev`.
+* After you've made your changes, please run ESLint(`npm run lint:fix`) and make sure your commits abide by the Conventional Commits rule as described in `CONTRIBUTING.md`. You can do this automatically by using the git hooks in this repo using husky.
```

Comment:
Add space before `(` and change several lines to make it more intuitive like the suggestion below

```suggestion
* Have NodeJS installed *(obviously)* and clone this repo
* Copy `.env.example` to `.env.local` and fill all the values
* Now, you can now start coding! Start the dev server by running `npm run dev`.
* After you've made your changes, please run ESLint by running `npm run lint:fix` and make sure your commits abide by the Conventional Commits rule as described in [`CONTRIBUTING.md`](https://github.com/GoldElysium/hefs-website/blob/master/CONTRIBUTING.md). You can do this automatically by using the git hooks in this repo using [Husky](https://github.com/typicode/husky).
```